core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -114,7 +114,9 @@ object ApiVersion {
     // Introduced topic IDs to LeaderAndIsr and UpdateMetadata requests/responses (KIP-516)
     KAFKA_2_8_IV1,
     // Introduce AllocateProducerIds (KIP-730)
-    KAFKA_3_0_IV0
+    KAFKA_3_0_IV0,
+    // Introduce ListOffsets maxTimestamps (KIP-734)
```

Comment:
Could we say `Introduce ListOffsets V7 which supports listing offsets by max timestamp (KIP-734)`.

Reply:
done